tox.ini

```diff
@@ -1,7 +1,6 @@
 [tox]
-envlist = py27
+envlist = py{27,34}
```

Comment:
We should also add python 3 test run to .travis.yml

    python:
      - "2.7"
      - "3.4"

    deploy:
      on:
        python: 3.4